helm_deploy/hmpps-offender-categorisation-api/values.yaml

```diff
@@ -64,3 +64,4 @@ generic-prometheus-alerts:
 cron:
   report_released_prisoners_with_active_categorisations_schedule: "0 6 1 * *"
   report_prs_eligibility_investigation_schedule: "0 0 1 * *"
+  poll_prisoners_risk_profile_schedule: "15 2 * * 1-5"
```

Comment:
Nothing wrong here, just wondered why Mon-Fri only? 
Is this just replication of the risk profilers schedule? 🤔